### PR TITLE
Support IPinfo database files in GeoIP lookup data adapter

### PR DESCRIPTION
**Pull request by bernd**
Add support for the standard location and ASN IPinfo database files.

### CODE REVIEW

File: graylog2-server/src/main/java/org/graylog/plugins/map/geoip/MaxmindDataAdapter.java [hunk 1]
```diff
@@ -245,6 +254,55 @@ protected LookupResult doGet(Object key) {
...
+                }
+            case IPINFO_STANDARD_LOCATION:
+                try {
+                    final IPinfoStandardLocation ipInfo = reader.ipInfoStandardLocation(addr);
+                    if (ipInfo == null) {
+                        LOG.debug("No IPinfo location data for IP address {}, returning empty result.", addr);
+                        return LookupResult.empty();
+                    }
+
+                    return LookupResult.multi(ipInfo.coordinates(), ImmutableMap.builder()
```
Comment (thll) [inline graylog2-server/src/main/java/org/graylog/plugins/map/geoip/MaxmindDataAdapter.java:289]: Can we be sure that we get data for all these fields back from the DB? E.g. is `city` guaranteed to be always provided?  I couldn't find an example for incomplete data, but what do you think about being a bit more cautious? We could make fields in `IPinfoStandardLocation` nullable/optional and construct the map with only those values which are present here.

**Review by thll — CHANGES_REQUESTED** [review]
Looks good overall and I couldn't find any issues when testing this.

Found just two minor things worth taking care of.

File: graylog2-server/src/test/java/org/graylog/plugins/map/geoip/MaxmindDataAdapterTest.java [hunk 2]
```diff
@@ -41,18 +40,24 @@
 
 @RunWith(Suite.class)
 @Suite.SuiteClasses({
-        MaxmindDataAdapterTest.CityDatabaseTest.class,
-        MaxmindDataAdapterTest.CountryDatabaseTest.class
-})
+                            MaxmindDataAdapterTest.CityDatabaseTest.class,
+                            MaxmindDataAdapterTest.CountryDatabaseTest.class,
+                            MaxmindDataAdapterTest.AsnDatabaseTest.class,
+                            MaxmindDataAdapterTest.IPinfoStandardLocationDatabaseTest.class,
```
[the remark this answers is not in the document]
Comment (bernd) [reply graylog2-server/src/test/java/org/graylog/plugins/map/geoip/MaxmindDataAdapterTest.java:46]: Good catch, thank you!

File: graylog2-server/src/main/java/org/graylog/plugins/map/geoip/MaxmindDataAdapter.java (hunk 1 above)
Comment (bernd) [reply graylog2-server/src/main/java/org/graylog/plugins/map/geoip/MaxmindDataAdapter.java]: Aye, I made all properties nullable. :+1:

**Review by thll — APPROVED** [review]
LGTM

**Pull request merged**